geode-docs/tools_modules/geode_apis_compatible_with_redis.html.md.erb

```diff
@@ -64,52 +64,65 @@ If the server is functioning properly, you should see a response of `PONG`.
+The <%=vars.product_name%> APIs compatible with Redis currently supports the following Redis commands.
+- APPEND
+- AUTH
+- DECR
+- DECRBY
+- DEL
+- EXISTS
+- EXPIRE
+- EXPIREAT
+- GET
+- GETRANGE
+- HDEL
+- HEXISTS
+- HGET
+- HGETALL
+- HINCRBY
+- HINCRBYFLOAT
+- HLEN
+- HMGET
+- HMSET
+- HSCAN
+- HSET
+- HSETNX
+- HSTRLEN
+- HVALS
+- HKEYS
+- INCR
+- INCRBY
+- INCRBYFLOAT
+- INFO
+- KEYS
+- MGET
+- PERSIST
+- PEXPIRE
+- PEXPIREAT
+- PING
+- PSUBSCRIBE
+- PTTL
+- PUBLISH
+- PUNSUBSCRIBE
+- QUIT
+- RENAME
+- SADD
+- SET
+- SETNX
+- SLOWLOG <sup>1</sup>
+- SMEMBERS
+- SREM
+- STRLEN
+- SUBSCRIBE
+- TTL
+- TYPE
+- UNSUBSCRIBE
 
-**Note**: These commands are supported for Redis 5.
-**Note**: SLOWLOG is implemented as a NoOp for compatibility with some monitoring tools.
-
--   **Connection**: AUTH, PING, QUIT
--   **Hashes**: HDEL, HEXISTS, HGET, HGETALL, HINCRBY, HINCRBYFLOAT, HKEYS, HLEN, HMGET, HMSET, HSCAN, HSET, HSETNX, HSTRLEN, HVALS
--   **Keys**: DEL, EXISTS, EXPIRE, EXPIREAT, KEYS, PERSIST, PEXPIRE, PEXPIREAT, PTTL, RENAME, TTL,
-  TYPE
--   **Publish/Subscribe**: PUBLISH, PSUBSCRIBE, PUNSUBSCRIBE, SUBSCRIBE, UNSUBSCRIBE
--   **Server**: SLOWLOG, INFO
--   **Sets**: SADD, SMEMBERS, SREM
--   **Strings**: APPEND, DECR, DECRBY, GET, GETRANGE, INCR, INCRBY, INCRBYFLOAT, MGET, SET, SETNX, STRLEN
-
-The following Redis commands are **unsupported** by <%=vars.produce_name%>'s APIs compatible with Redis. Unsupported
-commands are available to use, but have not been fully tested. There is no guarantee they will work
-exactly as expected.
-
--   **Connection**: ECHO, SELECT
--   **Keys**: SCAN, UNLINK
--   **Server**: DBSIZE, FLUSHALL (no async option), FLUSHDB (no async option), SHUTDOWN, TIME
--   **Sets**: SCARD, SDIFF, SDIFFSTORE, SINTER, SINTERSTORE, SISMEMBER, SMOVE, SPOP, SRANDMEMBER,
-  SSCAN, SUNION, SUNIONSTORE
--   **Strings**: BITCOUNT, BITOP, BITPOS, GETBIT, GETSET, MSET, MSETNX, PSETEX, SETBIT, SETEX, SETRANGE
-
-
-If you already have some Geode servers running that are compatible with Redis, you can execute the
-following command with gfsh to enable unsupported commands:
-
-```pre
-redis --enable-unsupported-commands
-```
-
-You can also enable unsupported commands when you start the Geode server by setting the Java property `enable-redis-unsupported-commands=true`:
+Commands not listed above are **not implemented**.
 
-```pre
-start server \
-  --J=-Denable-redis-unsupported-commands=true \
-  --name=<serverName> \
-  --locators=<locatorPort> \
-  --compatible-with-redis-port=<compatibleWithRedisPort> \
-  --compatible-with-redis-bind-address=<compatibleWithRedisBindAddress> \
-  --compatible-with-redis-password=<compatibleWithRedisPassword>
-```
+**Note**: These commands are supported for Redis 5.
 
-Commands not listed above are **not implemented**.
+**Note**: SLOWLOG is implemented as a NoOp for compatibility with some monitoring tools.
```

Comment:
based on our synchronous discussion:
one solution that would allow us to put the notes inline with the commands would be to bold the commands and have the note as a line or paragraph after the bolded text. so it would look like:

- **SETNX**
- **SLOWLOG** - SLOWLOG is implemented as a NoOp for compatibility with some monitoring tools
- **SMEMBERS**

Reply:
as per convo, lets wait to see how long info shakes out to be when all reviews are in, and then see if @Kmiller thinks they should all be in-lined, or all be footnotes, or different approaches for different commands/length of notes